sdks/java/io/rabbitmq/src/main/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqMessage.java

```diff
@@ -78,30 +80,37 @@ private static Map<String, Object> serializableHeaders(Map<String, Object> heade
     if (headers != null) {
       for (Map.Entry<String, Object> h : headers.entrySet()) {
         Object value = h.getValue();
-        if (!(value instanceof Serializable)) {
-          try {
-            if (value instanceof LongString) {
-              LongString longString = (LongString) value;
```

Comment:
I'm not sure why this was splitted to 4 lines. Persumably this makes error stack trace, if happens, more obvious which operation has problem?

Reply:
Fixed